psst-gui/src/controller/playback.rs

```diff
@@ -222,6 +224,11 @@ impl PlaybackController {
         self.send(PlayerEvent::Command(PlayerCommand::Seek { position }));
     }
 
+    fn update_volume(&mut self, volume: f64)  {
+        self.send(PlayerEvent::Command(PlayerCommand::SetVolume { volume: ((volume / 100.0) as f32) }));
```

Comment:
The slider widget implements Widget<f64> and I then convert it to f32 for miniaudio

Reply:
Also we have to divide by 100 as the values is chosen in 0 - 100 but miniaudio is expecting 0 - 1